packages/create-gatsby/src/index.ts

```diff
@@ -167,6 +174,23 @@ ${center(c.blueBright.bold.underline(`Welcome to Gatsby!`))}
 
   const data = await enquirer.prompt(questions)
 
+  trackCli(`CREATE_GATSBY_SELECT_OPTION`, {
+    name: `project_name`,
+    valueString: sha256(data.project),
+  })
+  trackCli(`CREATE_GATSBY_SELECT_OPTION`, {
```

Comment:
Can we capture these when we get the response, not at once when they have given all answers?
That way we can tell how long it took to answer each.

Reply:
With the actual implementation, we got all the answers at once (the form is managed through an array). While we can find ways to extract each input in its own and dedicated form that we can track, it would require some time to refactor.

I think that this product is supposed to get stable in the week, so what do you think to create an issue for handling the more granular scenario but to merge this one in that state?

cc @ascorbic @jamo

Reply:
That's ok. It's most important to know what they pick, even if we can't get timings. (do we get anything if they `ctrl+c` while in the form?)

Reply:
It will track every event that has been answered but will end with `CREATE_GATSBY_END` exit code 0